### PR TITLE
Issue templates for org-wide repo

### PR DESCRIPTION
**Pull request by joshcanhelp**
## Description

Merge in feedback from existing template here and the [Community getting started](https://github.com/auth0-community/getting-started/blob/master/ISSUE_TEMPLATE.md). This is in preparation for creating an org-wide community health file repo. 

## References

- [Org-wide community health files](https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/)
- [Auth0 internal project page](https://auth0team.atlassian.net/wiki/spaces/~783316165/pages/251726712/Org-wide+community+health+files)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+about: Create a bug report to fix an existing issue.
+title: ''
+labels: ''
+assignees: ''
+
+---
+**Thank you in advance for helping us to improve this library!** Please read through the template below and answer all relevant questions. Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, please use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/).
+
+By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
+
```
Comment (mosspilot) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Somewhere in here we should remind them to search if the issue already exists so we don't get a bunch of duplicates.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+about: Suggest new functionality for this project.
+title: ''
+labels: ''
+assignees: ''
+
+---
+**Thank you in advance for helping us to improve this library!** Your attention to detail here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, please use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/).
+
+By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
+
```
Comment (mosspilot) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Same comment about checking if the issue has already been reported

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+name: Report an issue
+about: Create a bug report to fix an existing issue.
+title: ''
+labels: ''
+assignees: ''
+
+---
+**Thank you in advance for helping us to improve this library!** Please read through the template below and answer all relevant questions. Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, please use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/).
+
```
Comment (mosspilot) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Somewhere in here, in big bold letters we should tell them not to report vulnerabilities through here and use our responsible disclosure system.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (mosspilot) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Same comment about vulnerability disclosure

**Comment by beardaway** [comment]
Totally agree!

**Pull request merged**